website/static/js/formViewModel.js

```diff
@@ -0,0 +1,45 @@
+/*
+ * Maintains the base class for knockoutJS form ViewModels
+ */
+'use strict';
+
+var ko = require('knockout');
+
+var $osf = require('osfHelpers');
+
+
+var ValidationError = function(errorHeader, errorMessage) {
```

Comment:
`ValidationError` should subclass `Error` (again, you can use the `oop` module).